sklearn/neighbors/_base.py

```diff
@@ -521,7 +521,6 @@ def _fit(self, X, y=None):
         self._fit_method = self.algorithm
         self._fit_X = X
         self.n_samples_fit_ = X.shape[0]
-
```

Comment:
This line removal is not related. May you revert this?

(It makes it easier to run `git blame`.)